internal/comment/github.go

```diff
@@ -113,6 +113,10 @@ func newGitHubAPIClients(ctx context.Context, token string, apiURL string, tlsCo
 	if !strings.HasSuffix(u.Path, "/") {
 		u.Path += "/"
 	}
+	// Check if "/api/v3/" already exists in the URL path.
+	if strings.Contains(u.Path, "/api/v3/") {
```

Comment:
```suggestion
	if strings.HasSuffix(u.Path, "/api/v3/") {
```
We could use `HasSuffix` here to make sure we are only checking the end of the string.

Reply:
```go
	// Check if "/api/v3/" already exists in the URL path.
	if strings.HasSuffix(u.Path, "/api/v3/") {
		return nil, nil, fmt.Errorf("the /api/v3/ suffix should not be included in the --github-api-url")
	}
```
After this, we also have to lowercase the first **The** as it will show an error **error string should not be capitalized**